src/lib/rest/rest.cpp

```diff
@@ -518,6 +518,18 @@ static char* removeTrailingSlash(std::string path)
 */
 int servicePathSplit(ConnectionInfo* ciP)
 {
+  //
+  // Special case: empty service-path 
+  //
+  if ((ciP->httpHeaders.servicePathReceived == true) && (ciP->servicePath == ""))
```

Comment:
Discussed by skype: let's try to figure out if removing this block of code is fixing something and, in that case, include a .test harness to cover the case and an issue to be added to CHANGES_NEXT_RELEASE.